Makefile

```diff
@@ -91,6 +91,7 @@ create-install-files:
 	IMG=$(NS)$(IMAGE_NAME_OPERATOR):$${IMAGE_TAG} \
 	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
 	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
+	LOG_LEVEL=error \

```

Comment:
error log level will be very sparse. Recommend this to be LOG_LEVEL=info 

We shouldn't be afraid to log important informational events. Errors only will be lacking context during analysis, is my guess

Reply:
great catch, will change, thanks!

Reply:
great catch, thanks!